lib/realtime/tenants/cache.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Realtime.Tenants.Cache do
+  @moduledoc """
+  Cache for users.
+  """
+
+  require Cachex.Spec
+
+  alias Realtime.Tenants
+
+  def child_spec(_) do
+    %{
+      id: __MODULE__,
+      start:
+        {Cachex, :start_link, [__MODULE__, [expiration: Cachex.Spec.expiration(default: 30_000)]]}
```

Comment:
cache invalidation works on each node independently, and this can lead to inconsistency across the cluster

Reply:
v2 of this will listen to the wal, and invalidate the cache.